content/issues/2023-02-14-running-grouping-migrations.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Running grouping migrations'
+date: '2023-02-14T02:00:00+01:00'
+affected:
+  - API
+  - Workers
+  - Dashboard
+  - Copr
+  - Testing Farm
+  - Koji
+resolved: false
+resolvedWhen:
+section: issue
+severity: down
+---
+
+We are updating our database schema to facilitate further development.
+As a part of this process, multiple database migrations will have to
+be run. The first step is grouping together related builds and test
+runs. The migrations will happen during the next deployment to production,
+on Tuesday February 14 at 2:00AM CET. This set of migrations is expected to
```

Comment:
Users will get a Valentine's gift... :smile:

Reply:
Yeah, that was my immediate thought when I was checking the calendar - breaking the prod on Valentine's day :D